packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1419,6 +1426,11 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Null by default.
   final ButtonStyle? style;
 
+  /// Whether to request focus for it when the menu appears.
+  ///
+  /// Defaults is [Navigator.requestFocus].
```

Comment:
This is not strictly true. Instead maybe say: "If null, [Navigator.requestFocus] will be used instead."

Here and in a few other places where you document requestFocus.